tdesign/src/Icon/cssSvgStrings.ts

```diff
@@ -170,6 +170,8 @@ export const passwordHideURI =
   "data:image/svg+xml;charset=utf8,%3Csvg width='20' height='20' viewBox='0 0 20 20' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M9.32644 16.2273C9.549 16.2423 9.77359 16.25 9.99998 16.25C14.1666 16.25 17.725 13.6583 19.1666 10C18.1666 7.46226 16.148 5.4378 13.6143 4.43568M7.5 15.9293C4.44867 15.1294 1.97605 12.8998 0.833313 10C2.27498 6.34167 5.83331 3.75 9.99998 3.75C10.5946 3.75 11.1769 3.80279 11.7424 3.90394' stroke='%23555656' stroke-width='1.25'/%3E%3Cpath d='M12.9167 0.833313L6.5 19' stroke='%23555656' stroke-width='1.25'/%3E%3C/svg%3E";
 export const pencilBlueIconURI =
   "data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='none' stroke='%2336678d' stroke-width='2' stroke-linecap='round' stroke-linejoin='round' class='feather feather-edit-2'%3E%3Cpath d='M17 3a2.828 2.828 0 1 1 4 4L7.5 20.5 2 22l1.5-5.5L17 3z'%3E%3C/path%3E%3C/svg%3E";
+export const pencilIconURI =
```

Comment:
General question about the patterns we use in the codebase, again, to @milesrichardson:

What is the value of having a single `cssSvgStrings` file with data URLs of all icons, compared to storing those data URLs in individual icon component files (`IconPencil.tsx` in this case)?

I see we are doing that at the moment, but IMO moving those data URL strings to component files makes more sense, since that results in more cohesion inside modules (modules are self-sufficient) and reduces indirection

Reply:
The only real advantage is that it means we can import the CSS string itself directly, which can be useful in the rare case of using it outside the icon component (e.g., using it as a value in a CSS `background-image` property). We do this a few times.

That said, the same could be accomplished by simply colocating the string with its component. 

I think the original reason was because it made for the easiest workflow of code generation. I would get the SVG string, put it here, and then run `mkrci` to create the component referencing the name of the string I just defined in this file.

I'm going to modify that clickup I just mentioned in the above comment to be a more general "refactor icon system"...

Reply:
Sounds good!